src/worker/tasks/processTx.ts

```diff
@@ -355,16 +295,12 @@ export const processTx = async () => {
                   txIndex++;
 
                   rpcResponses.push({
-                    queueId: tx.queueId!,
-                    tx: txRequest,
-                    res: rpcResponse,
-                    sentAt: new Date(),
-                    functionName: tx.functionName || undefined,
-                    extension: tx.extension || undefined,
-                    queuedAt: new Date(tx.queuedAt!).getTime(),
```

Comment:
Most of these fields are from `tx`, the DB transaction object. I'm adding tx to the payload to simplify readability. Now we're passing around `(tx db object, ethers tx request, rpc response)`.